File: packages/tasks-gen/snippets-fixtures/explicit-direct-request/js/fetch/0.together.js
async function query(data) {
	const response = await fetch(
		"https://api.together.xyz/v1/chat/completions",
		{
			headers: {
				Authorization: `Bearer ${process.env.TOGETHER_API_KEY}`,
				"Content-Type": "application/json",
			},
			method: "POST",
			body: JSON.stringify(data),
		}
	);
	const result = await response.json();
	return result;
}

query({ 
    messages: [
        {
            role: "user",
            content: "What is the capital of France?",
        },
    ],
    model: "<together alias for meta-llama/Llama-3.1-8B-Instruct>",
}).then((response) => {
    console.log(JSON.stringify(response));
});

File: packages/tasks-gen/snippets-fixtures/explicit-direct-request/python/requests/0.together.py
import os
import requests

API_URL = "https://api.together.xyz/v1/chat/completions"
headers = {
    "Authorization": f"Bearer {os.environ['TOGETHER_API_KEY']}",
}

def query(payload):
    response = requests.post(API_URL, headers=headers, json=payload)
    return response.json()

response = query({
    "model": "<together alias for meta-llama/Llama-3.1-8B-Instruct>",
    "messages": [
        {
            "role": "user",
            "content": "What is the capital of France?"
        }
    ]
})

print(response["choices"][0]["message"])